idaes/apps/grid_integration/examples/thermal_generator.py

```diff
@@ -139,6 +139,7 @@ def populate_model(self, b, segment_number = 4):
         '''
 
         model_data = self.model_data
+        segment_number = 4
```

Comment:
Set this in the __init__ method?

Reply:
Thanks, @adowling2 ! Because this class is for RTS-GMLC and the number of segments is always 4, I made `segment_number` a class attribute.

Reply:
Thank you; I added a comment to clarify this.